var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,29 +34,46 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
-    version('develop')
-    version('2.1', tag='2.1')
-    version('2.0.3', tag='2.0.3')
+    version('3.0_beta',
+            git='https://bitbucket.org/scheres/relion-3.0_beta.git')
+    version('2.1', git='https://github.com/3dem/relion.git', preferred='true')
+    version('2.0.3', git='https://github.com/3dem/relion.git')
+    version('develop', git='https://github.com/3dem/relion.git')
```

Comment:
Version 3.0_beta is the only one that needs to specify `git=`. The rest can use the out package-level git.

Reply:
I am a bit confused since  `https://bitbucket.org/scheres/relion-3.0_beta.git` does not contain code for previous versions.

Version 2 and lower are at `https://github.com/3dem/relion.git`

Reply:
Oh, I see!

`git = https://github.com/3dem/relion.git` has previous versions.